keywords/TestServeriOS.py

```diff
@@ -28,16 +27,29 @@ def __init__(self, version_build, host, port, community_enabled=None, debug_mode
         self.debug_mode = False
         self.app_path = ""
         self.app_name = ""
+        self.platform = platform
+        self.bundle_id = ""
+
         if debug_mode:
             self.debug_mode = True
-        if community_enabled:
-            self.app_dir = "CBLTestServer-iOS-community-{}".format(version_build)
-            self.package_name = "CBLTestServer-iOS-community-{}.zip".format(version_build)
-            self.app = "CBLTestServer-iOS"
+
+        if self.platform == "ios":
+            if community_enabled:
+                self.app_dir = "CBLTestServer-iOS-community-{}".format(version_build)
+                self.package_name = "CBLTestServer-iOS-community-{}.zip".format(version_build)
+                self.app = "CBLTestServer-iOS"
+            else:
+                self.app_dir = "CBLTestServer-iOS-enterprise-{}".format(version_build)
+                self.package_name = "CBLTestServer-iOS-enterprise-{}.zip".format(version_build)
+                self.app = "CBLTestServer-iOS-EE"
+
+            self.bundle_id = "com.couchbase.CBLTestServer-iOS"
         else:
```

Comment:
not covered for community edition and Enterprise edition condition? I think it's ok as we are testing only enterprise edition here

Reply:
There is no CE app. Jim said that we will support CE version but will not carry out the same level of testing, so no need for a CE version of the app